src/ir_Mitsubishi.h

```diff
@@ -58,6 +60,8 @@ const uint8_t kMitsubishiAcStartStopTimer = 7;
 #define MITSUBISHI_AC_DRY kMitsubishiAcDry
 #define MITSUBISHI_AC_COOL kMitsubishiAcCool
 #define MITSUBISHI_AC_AUTO kMitsubishiAcAuto
+#define MITSUBISHI_AC_WIDE_VANE_AUTO kMitsubishiAcWideVaneAuto
```

Comment:
Remove this line too. We don't support adding more `#define`s anymore.

Reply:
Ok No problem